grpc/tests/grpctest.cpp

```diff
@@ -55,17 +52,16 @@ class ServiceImpl final : public MyGame::Example::MonsterStorage::Service {
          request->GetRoot()->id()->str() + " No." + std::to_string(i)));
        fbb_.Finish(monster_offset);
 
-       flatbuffers::BufferRef<Monster> monsterRef(
-         fbb_.GetBufferPointer(), fbb_.GetSize()
-       );
+       flatbuffers::grpc::Message<Monster> monster = fbb_.ReleaseMessage<Monster>();
+
        // Send monster to client using streaming.
-       writer->Write(monsterRef);
+       writer->Write(monster);
      }
      return grpc::Status::OK;
   }
 
  private:
-  flatbuffers::FlatBufferBuilder fbb_;
+  flatbuffers::grpc::MessageBuilder fbb_;
```

Comment:
Yes, I agree that we should beef up documentation on this. I'll contribute what I can with this PR, but we'll probably have to keep expanding over time. I hope to see gRPC + flatbuffers as well documented as the default protobuf usage.